web/src/main/webapp/dataViews/schemaView.ts

```diff
@@ -238,6 +245,14 @@ export class SchemaView extends TSViewBase {
                     data.push("", "", "", "", "");
                 }
             }
+            if (this.counts != null) {
```

Comment:
Why would it be missing?

Reply:
When a table has just been loaded, the `stats` and `counts` are null because no sketches have been run. It will display the schema view but the stats and distinct count column will be hidden.